sdk/python/setup.py

```diff
@@ -48,6 +48,7 @@
     "numpy",
     "google",
     "confluent_kafka",
+    "tensorflow-metadata>=0.21,<0.22",
```

Comment:
What is this for?

Reply:
This is a missing dependency because we are using the TensorFlow metadata proto

Previously I keep facing this bug whenever I tried to generate my proto. We verified on a clean install on @davidheryanto environment as well

Reply:
Ok, but how is it related to your PR? Should it be a separate PR?

Reply:
I think you're right, should created a separate PR
https://github.com/gojek/feast/pull/508

This is the PR @woop, this also generates the python SDK Protos

This will help to reduce the file change in this PR, so we can focus on the features